src/Command/Proxy/ApplicationConnectionHelper.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Doctrine\Bundle\DBALBundle\Command\Proxy;
+
+use Doctrine\DBAL\Tools\Console\Helper\ConnectionHelper;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+trait ApplicationConnectionHelper
+{
+    private function setConnectionHelper(Application $application, ?string $connectionName)
+    {
+        $connection = $application->getKernel()->getContainer()->get('doctrine.dbal.connection_registry')->getConnection($connectionName);
```

Comment:
@stof from our discussion at the SymfonyHackday I remember that you mentioned that with the ConnectionRegistry added to doctrine/dbal this could be simplified, right?

But for now we should add all the proxy commands from DoctrineBundle, right? As this has to be refactored then in doctrine/dbal first I guess?